workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerRegistry.cs

```diff
@@ -37,37 +36,33 @@ public static void UnsetWorkerForWorld(WorkerBase worker, World world)
 
         public static WorkerBase GetWorkerForWorld(World world)
         {
-            WorkerBase worker;
-            WorldToWorker.TryGetValue(world, out worker);
+            WorldToWorker.TryGetValue(world, out var worker);
             return worker;
         }
 
         public static void RegisterWorkerType<T>() where T : WorkerBase
         {
-            string workerType = (string) typeof(T).GetField("WorkerType").GetValue(null);
+            WorkerTypes.Add(typeof(T).Name, typeof(T));
             WorkerTypeToAttributeSet.Add(
                 typeof(T),
-                new WorkerAttributeSet(new Improbable.Collections.List<string> { workerType })
+                new WorkerAttributeSet(new Improbable.Collections.List<string> { typeof(T).Name })
```

Comment:
`nameof(T)`

Reply:
`nameof`doesn't work with generics. it will print `T`instead of e.g. `UnityClient`